src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -267,28 +272,37 @@ export function* lcManager(params={}) {
     }
 }
 
-function updateRawTableChart(timeCName, fluxCName) {
+
+function updateRawTableChart(timeCName, fluxCName, mission='') {
     var chartX = get(getChartDataElement(LC.RAW_TABLE), ['options', 'x', 'columnOrExpr']);
     var chartY = get(getChartDataElement(LC.RAW_TABLE), ['options', 'y', 'columnOrExpr']);
 
     if (chartX === timeCName && chartY === fluxCName) return;
 
     if (timeCName && fluxCName) {
-        const xyPlotParams = {x: {columnOrExpr: timeCName}, y: {columnOrExpr: fluxCName, options: 'grid,flip'}};
+
+        const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
```

Comment:
why is 'wise' hard-coded here?  it should not.  this logic needs to be moved into the converter.  that's why they are there.  this applies to all of the 'wise' and 'other' references in this file.